packages/camera/camera_windows/windows/capture_controller.cpp

```diff
@@ -288,17 +288,19 @@ void CaptureControllerImpl::ResetCaptureController() {
   texture_handler_ = nullptr;
 }
 
-void CaptureControllerImpl::InitCaptureDevice(
+bool CaptureControllerImpl::InitCaptureDevice(
     flutter::TextureRegistrar* texture_registrar, const std::string& device_id,
     bool record_audio, ResolutionPreset resolution_preset) {
   assert(capture_controller_listener_);
 
   if (IsInitialized()) {
-    return capture_controller_listener_->OnCreateCaptureEngineFailed(
+    capture_controller_listener_->OnCreateCaptureEngineFailed(
         "Capture device already initialized");
+    return false;
   } else if (capture_engine_state_ == CaptureEngineState::kInitializing) {
-    return capture_controller_listener_->OnCreateCaptureEngineFailed(
+    capture_controller_listener_->OnCreateCaptureEngineFailed(
```

Comment:
Should these conditions be combined? They look identical

Reply:
The error message is slightly different (`initialized` vs `initializing`).